CMakePresets.json

```diff
@@ -52,6 +52,110 @@
                 "value": "x86"
             }
         },
+        {
```

Comment:
Please revert the changes to this file. (Fine to have them locally, of course!)

Reply:
Yes its the whole idea of CMakeUserPresets.json. if you just copy paste this into that file it will all work fine.